### PR TITLE
Adjusted python2-cherrypy naming in salt-api.

### PR DESCRIPTION
**Pull request by sbluhm**
Fixed Cherrypy package name for Red Hat based systems.

### CODE REVIEW

File: salt/salt.spec [hunk 1]
```diff
@@ -691,8 +691,12 @@ Group:          System/Management
 Requires:       %{name} = %{version}-%{release}
 Requires:       %{name}-master = %{version}-%{release}
 %if 0%{?default_py3}
+%if 0%{?suse_version}
 Requires:       python3-CherryPy >= 3.2.2
 %else
+Requires:       python3-cherrypy
+%endif
+%else
 Requires:       python-CherryPy >= 3.2.2
```
Comment (brejoc) [inline salt/salt.spec]: I think we should have the same if-else for Python2. Could you add that please? Thanks!

**Review by brejoc — CHANGES_REQUESTED** [review]
Thanks a lot for the PR! One nitpick. I think it makes sense to have that for Python2 too, since the package was also named `python-cherrypy` in CentOS/RHEL 7.

File: salt/salt.spec (hunk 1 above)
Comment (sbluhm) [reply salt/salt.spec]: I have integrated the if statements in the top to avoid cluttering. The top if might also need some optimisation at some point.

**Review by brejoc — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**